orderbook/openapi.yml

```diff
@@ -486,6 +494,12 @@ components:
           $ref: "#/components/schemas/Signature"
         signingScheme:
           $ref: "#/components/schemas/SigningScheme"
+        balanceFrom:
+          $ref: "#/components/schemas/BalanceFrom"
+          default: "owner"
+        balanceTo:
+          $ref: "#/components/schemas/BalanceTo"
+          default: "owner"
```

Comment:
I get what this means and it's talking about the Balancer integration. 
But how does this play out with the `receiver` field?

If I set `balanceTo=owner` and `receiver=<not source(owner) address>` what happens?
What about when `balanceTo=vault_internal`?

I expect `receiver` would take precedence. I also assume `balanceTo` only applies to Balancer vaults.

Can you clarify?

Reply:
It would send to the receiver and not the owner. This enum represents how the trade proceeds get paid out, either in ERC20 tokens, or in Balancer Vault internal balances.

Reply:
And as an additional point, there is nothing preventing a transfer to a receiver's internal vault balance even if they have never used the balancer protocol.

Reply:
Now I'm more confused as to how receiver and balanceTo interact.
Are they completely independent of each other?

Reply:
BalanceTo will be renamed to BuyTokenBalance and it is just an enum indicating the location where the proceeds are to be transferred. receiver will always be the receiver of the proceeds, but this enum indicates where the receivers balance is to be updated. 

That is either the receivers Erc20 token balance is updated, or their internal vault balance.

Reply:
Synced over slack, all good now